crates/ruff_python_ast/src/comparable.rs

```diff
@@ -1604,3 +1700,402 @@ impl<'a> From<&'a ast::ModExpression> for ComparableModExpression<'a> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use ruff_text_size::TextRange;
+
+    use crate as ast;
+    #[test]
+    fn compare_concatenated_string_to_value() {
```

Comment:
Thanks for taking the time to write all those tests. 

I don't have a great suggestion but I wonder if we should make them integration tests instead, because they're very verbose and are difficult to read. E.g. there's so much boilerplate that I find it difficult to pick up the important bits. I also assume that they were very annoying to write. What do you think?

Reply:
The integration tests have the same content so I elected to remove these verbose unit tests entirely. They were helpful for development (and for my own understanding of the internals of f-string nodes), but keeping them around is probably more trouble than they're worth!